src/Stratis.Bitcoin.Features.LightWallet/Settings.StyleCop

```diff
@@ -1,796 +0,0 @@
-<StyleCopSettings Version="105">
```

Comment:
So how does work then? Where did the stylecop settings for this move to?

(Sorry the diff is a bit big :P)